zk.js/src/account.ts

```diff
@@ -286,18 +303,17 @@ export class Account {
     merkleTreePdaPublicKey: PublicKey,
     commitment: Uint8Array,
   ): Uint8Array {
-    const encryptedBytes = this.wasmAccount.encryptAesUtxo(
+    return this.wasmAccount.encryptAesUtxo(
       messageBytes,
       merkleTreePdaPublicKey.toBytes(),
       commitment,
     );
-    return encryptedBytes;
   }
 
   /**
    * Encrypts bytes with aes secret key.
    * @param messageBytes - The bytes to be encrypted.
-   * @param iv16 - Optional Initialization Vector (iv), 16 random bytes by default.
+   * @param iv12 - Optional Initialization Vector (iv), 12 random bytes by default.
```

Comment:
Just noticed that we need to return the iv as well to ensure that we can decrypt the cipher text.
We can for example prepend the iv:
```
return [...iv12, ...this.wasmAccount.encryptAes(messageBytes, iv12)];
```

Reply:
We are doing it on Rust side here: https://github.com/Lightprotocol/light-protocol/blob/sergey/nacl2wasm/account.rs/src/wasm/src/account.rs#L323-L324

Reply:
I see thanks!